<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>711</width>
       <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                      <layout class="QVBoxLayout" name="verticalLayout">
                          <item>
                              <widget class="QRadioButton" name="radio_sh">
                                  <property name="maximumSize">
                                      <size>
                                          <width>160</width>
                                          <height>16777215</height>
                                      </size>
                                  </property>
                                  <property name="focusPolicy">
                                      <enum>Qt::ClickFocus</enum>
                                  </property>
                                  <property name="text">
                                      <string>Схема</string>
                                  </property>
                                  <attribute name="buttonGroup">
                                      <string notr="true">radios</string>
                                  </attribute>
                              </widget>
                          </item>
                          <item>
                              <widget class="QRadioButton" name="radio_sp">
                                  <property name="maximumSize">
                                      <size>
                                          <width>160</width>
                                          <height>16777215</height>
                                      </size>
                                  </property>
                                  <property name="focusPolicy">
                                      <enum>Qt::ClickFocus</enum>
                                  </property>
                                  <property name="text">
                                      <string>Спутник</string>
                                  </property>
                                  <attribute name="buttonGroup">
                                      <string notr="true">radios</string>
                                  </attribute>
                              </widget>
                          </item>
                          <item>
                              <widget class="QRadioButton" name="radio_gi">
                                  <property name="maximumSize">
                                      <size>
                                          <width>160</width>
                                          <height>16777215</height>
                                      </size>
                                  </property>
                                  <property name="focusPolicy">
                                      <enum>Qt::ClickFocus</enum>
                                  </property>
                                  <property name="text">
                                      <string>Гибрид</string>
                                  </property>
                                  <attribute name="buttonGroup">
                                      <string notr="true">radios</string>
                                  </attribute>
                              </widget>
                          </item>
                      </layout>
                  </item>
                  <item>
                      <widget class="QLabel" name="image">
                          <property name="minimumSize">
                              <size>
                                  <width>600</width>
                                  <height>450</height>
                              </size>
                          </property>
                          <property name="text">
                              <string/>
                          </property>
                      </widget>
                  </item>
              </layout>
          </item>
          <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                      <widget class="QPushButton" name="drop">
                          <property name="minimumSize">
                              <size>
                                  <width>50</width>
                                  <height>20</height>
                              </size>
                          </property>
                          <property name="text">
                              <string>Сброс</string>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <widget class="QLineEdit" name="search_line"/>
                  </item>
                  <item>
                      <widget class="QPushButton" name="search">
                          <property name="minimumSize">
                              <size>
                                  <width>50</width>
                                  <height>20</height>
                              </size>
                          </property>
                          <property name="text">
                              <string>Искать</string>
                          </property>
                      </widget>
                  </item>
              </layout>
    </item>
          <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                      <widget class="QLabel" name="label">
                          <property name="text">
                              <string>Полный адрес</string>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <widget class="QLineEdit" name="full_place">
                          <property name="readOnly">
                              <bool>true</bool>
                          </property>
                      </widget>
                  </item>
                  <item>
                      <widget class="QCheckBox" name="mail">
                          <property name="text">
                              <string>Почтовый адрес</string>
                          </property>
                      </widget>
                  </item>
              </layout>
          </item>
   </layout>
  </widget>
 </widget>
    <tabstops>
        <tabstop>radio_sh</tabstop>
        <tabstop>radio_sp</tabstop>
        <tabstop>radio_gi</tabstop>
        <tabstop>search_line</tabstop>
        <tabstop>search</tabstop>
    </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radios"/>
 </buttongroups>
</ui>
